pkg/syncer/metadatasyncer.go

```diff
@@ -1191,15 +1189,18 @@ func initResizeReconciler(ctx context.Context, tkgClient clientset.Interface, su
 		log.Errorf("resize: could not get supervisor namespace in which Tanzu Kubernetes Grid was deployed. Resize reconciler is not running for err: %v", err)
 		return err
 	}
-	onceForVolumeResizeReconciler.Do(func() {
-		log.Infof("initResizeReconciler is triggered")
-		informerFactory := informers.NewSharedInformerFactory(tkgClient, resizeResyncPeriod)
-
-		rc := newResizeReconciler(tkgClient, supervisorClient, supervisorNamespace, resizeResyncPeriod, informerFactory,
-			workqueue.NewItemExponentialFailureRateLimiter(resizeRetryIntervalStart, resizeRetryIntervalMax),
-		)
-		informerFactory.Start(wait.NeverStop)
-		rc.Run(ctx, resizeWorkers)
-	})
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+	log.Infof("initResizeReconciler is triggered")
+	informerFactory := informers.NewSharedInformerFactory(tkgClient, resizeResyncPeriod)
```

Comment:
Instead of creating a new factory instance, can we invoke https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/kubernetes/informers.go#L50? This way we can consolidate all the informers under one factory.

Reply:
We discussed this offline as well. Now that we have an issue filed, can we add the issue link here?

Reply:
Yes. This is the issue to track this https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/585